src/resources/postcss/tickets.pcss

```diff
@@ -27,6 +27,60 @@
 	}
 }
 
+/*
+	Tribe Horizontal Datepicker (Day, Month, Year)
+*/
+.tribe_horizontal_datepicker__container {
+	label {
+		width: 100%;
+		display: inline-block;
+	}
+	.tribe_horizontal_datepicker {
+		width: 100%;
+		margin: var(--spacer-1) 0;
+		display: inline-block;
+
+		@media(--viewport-medium) {
+			width: calc( ( 100% / 3 ) - var(--spacer-1) - 4px ); /* 4px = safety margin */
+			margin-right: var(--spacer-1);
+			display: inline-block;
+		}
+
+		select {
+			border-color: var(--form-color-border-secondary);
+			border-radius: var(--border-radius-default);
+			border-style: solid;
+			border-width: 1px;
+			width: 100%;
+
+			&:focus {
+				border-color: var(--color-accent-secondary-hover);
+			}
+
+			@media(--viewport-medium) {
+				max-width: 600px;
+			}
+
+			&.ticket-meta__has-error {
+				border-color: var(--color-icon-error) ! important;
+			}
+			@mixin mobile-body-2;
+			appearance: none;
+			background-color: var(--color-background);
+			background-image: url('../images/select.png');
+			background-position: right center;
+			background-repeat: no-repeat;
+			background-size: 33px 33px;
+			height: auto;
+			padding: 0.7em;
```

Comment:
I would not mix px and em, but rather stick to px

Reply:
done